internal/database/db.go

```diff
@@ -91,6 +94,32 @@ func (d *Datastore) initTables() error {
 		return err
 	}
 
+	// List of required columns
+	requiredColumns := map[string]string{
+		"suppress_messages": "BOOLEAN NOT NULL DEFAULT FALSE",
```

Comment:
I don't think you can make a new column NOT NULL without first assigning a value in the column to all existing rows. Could be wrong, but.

Reply:
Though, thinking more, I guess the DEFAULT value would come to save us here. I wasn't sure that applied to pre-existing rows but it appears it does.